source/docs/software/dashboards/shuffleboard/advanced-usage/shuffleboard-tuning-pid.rst

```diff
@@ -69,3 +69,19 @@ The following example demonstrates how to create a button on your dashboard that
       // and sends it to a motor
       motor.Set(pid.Calculate(encoder.GetDistance(), setpoint));
     }
+
+  .. code-block:: python
+
+    from wpilib.shuffleboard import Shuffleboard
+
+    tab = Shuffleboard.getTab("Shooter")
+    shooterEnable = tab.add("Shooter Enable", false).getEntry()
+
+    # Command Example assumed to be in a PIDSubsystem
+    NetworkButton(shooterEnable).onTrue(InstantCommand(m_shooter.enable()))
```

Comment:
Out of curiosity, have you tried running this? I don't think I ever got NetworkButton to work. Obviously not a python issue